scripts/chroot_build.sh

```diff
@@ -119,6 +118,10 @@ function install_pkg() {
     grub2-common \
     locales
     
+    if [ "$TARGET_UBUNTU_VERSION" != "jammy" ]; then
```

Comment:
concern: I think this would be better if we invert the predicate.  It's unlikely that future version of Ubuntu would re-introduce `lupin-casper` so better to only install it on specific platforms.  Less future maintenance.

Reply:
Thanks for your Feedback! I'll adjust this later today :)

Reply:
Only Take Care about bionic and focal? Or also cosmic, disco, eoan and groovy?

Reply:
Hmm, good question. @mvallim what versions of Ubuntu do we officially support?  Just LTS or every release?

Reply:
If he doesn't respond then probably best to just check for everything from bionic, which I believe is the earliest version supported by this project.

Reply:
Ive force pushed already checking dir focal and bionic https://github.com/mvallim/live-custom-ubuntu-from-scratch/compare/72a556f94db77bd39578c82f172fae5652409287..25993b69abd82285d11d406fd646595d3f605016

Maybe add more versions later if needed?  Bionic, focal and Jammy are those handled via GitHub actions.
The versions between also reached the end of official Support and i would need to know where the package Starts to be unavailable.

Reply:
Not Sure If interesting for your:

https://github.com/mvallim/live-custom-ubuntu-from-scratch/compare/master...andi34:live-custom-ubuntu-from-scratch:docker

Ive started to build via Docker, works fine so far. Based on pi-gen's Docker build process. Pi-gen ist released unser the `BSD 3-Clause "New" or "Revised" License`,  Not Sure If thats a Problem.

Reply:
Sounds reasonable to me @andi34 .  I'll give @mvallim some time to see this to see if he has anything to add.

IRT docker branch, nice!  I think that will help a lot of users that may get stuck w/ local env issues.  Very cool :+1: